app/screens/settings/notifications/notifications_disabled_notice/notifications_disabled_notice.tsx

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React, {useCallback, useMemo} from 'react';
+import {useIntl} from 'react-intl';
+import {StyleSheet, View} from 'react-native';
+import Permissions from 'react-native-permissions';
+
+import SectionNotice from '@components/section_notice';
+import {Screens} from '@constants';
+
+const styles = StyleSheet.create({
+    wrapper: {
+        flex: 1,
+        justifyContent: 'flex-end',
+        marginVertical: 16,
+    },
+});
+
+const NotificationsDisabledNotice = () => {
+    const intl = useIntl();
+
+    const onEnableNotificationClick = useCallback(async () => {
```

Comment:
We are not awaiting, do async is not needed

```suggestion
    const onEnableNotificationClick = useCallback(() => {
```